config/default.yml

```
@@ -5135,6 +5135,18 @@ Style/WordArray:
   # The regular expression `WordRegex` decides what is considered a word.
   WordRegex: !ruby/regexp '/\A(?:\p{Word}|\p{Word}-\p{Word}|\n|\t)+\z/'
 
+Style/YieldSelf:
+  Description: 'Preferred object methods.'
```

https://github.com/rubocop/ruby-style-guide/pull/895 has been merged. Can you insert the style guide link?
```suggestion
  Description: 'Preferred object methods.'
  StyleGuide: '#object-yield-self-vs-object-then'
```

---

And can you update text of `Description`?

---

Thank you for your review. I've added a link to the style guide and updated the `Description`.